### DISCUSSION TITLE
12/20

### DISCUSSION THREAD

**Discussion by oauch** [part3-4/Jawstify#32] [Daily_Meeting]
### 🔫 어제 한 일

기연님 : input, modal input 컴포넌트
경서님 : Button 컴포넌트
종민님 : Sidebar / 구성원 테이블 구현 중
소은님 : index 페이지
윤혁님 : DropDown 컴포넌트

### 남은 컴포넌트
- 카드
- 모달
- 테이블
<br />

### 컴포넌트 개발 기간
12월 23일 (토요일 오후 3시)
<br />

종민님 : 테이블 컴포넌트
기연님 : Modal input 부분
경서님 : 테이블 대시보드 컴포넌트
소은님 : 카드 컴포넌트
윤혁님 : 이미지 버튼 (로컬 이미지 불러오기)

<br />

### 🪓 오늘 할일

- 개발하기
- 갓생살기

<br />

### 💎 TMI
- 눈 천국 ❄️
<img src="https://github.com/part3-4/Taskify/assets/49686619/72c51bb9-8db9-482b-aa46-1ade4f4b514e" alt="눈" height="500" width="auto">
<br />

### 🪚 특이사항

- 다들 개발 화이팅 ⌨️

**Comment by summerkimm** [comment]
오 눈사람 만드셔야죠 ☃️
